modules/end-user-guide/pages/selecting-an-ide.adoc

```diff
@@ -5,13 +5,129 @@
 [id="selecting-an-ide_{context}"]
 = Selecting an IDE
 
-You can select your IDE at when starting a workspace: 
+You can select your preferred IDE when starting a workspace: 
+
+* link:https://theia-ide.org/docs/[Eclipse Theia], the current default editor
+* JetBrains IDE such as link:https://www.jetbrains.com/help/idea/getting-started.html[IntelliJ IDEA] or link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[PyCharm]
+* Visual Studio Code - link:https://github.com/Microsoft/vscode/[Open Source ("Code - OSS")]
+
+Currently, the latest stable Theia is the default editor when none is specified. 
+
+There are multiple ways to select an IDE when launching a workspace:
+
+* using the URL parameter `che-editor`
+* using a `.che/che-editor.yaml` file
+
+
+== Using URL parameters
+
+When launching a workspace using the factory API, you can pass in additional parameters so the chosen IDE is loaded.
```

Comment:
When launching a workspace using the factory API, you can pass in additional parameters to load the chosen IDE: